synapse/rest/client/sync.py

```diff
@@ -175,6 +175,11 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
         if since is not None:
             since_token = await StreamToken.from_string(self.store, since)
 
+        debug_token = parse_string(request, "synapse_debug_current_token")
```

Comment:
We could document the `?synapse_debug_current_token` query parameter under a new "Debugging" page under the "Development" section